pdr_backend/accuracy/app.py

```diff
@@ -350,12 +354,40 @@ def serve_statistics_from_file():
     If the file cannot be read or another error occurs, it returns a 500 Internal Server Error.
     """
 
+    four_weeks_ago = datetime.utcnow() - timedelta(weeks=4)
+    start_ts = UnixTimeS(int(four_weeks_ago.timestamp()))
```

Comment:
there are functions to get UnixTimeS from a datetime object. Please use those, instead of working with timestamps.

Reply:
if safe, you can also use the from natural language function

Reply:
again, not me... this should have been updated when the latest UnixTimeS or natural language functions were introduced

I have documented this in [#1424](https://github.com/oceanprotocol/pdr-backend/issues/1424)